src/main/java/com/google/devtools/build/lib/bazel/commands/ModCommand.java

```diff
@@ -499,11 +500,24 @@ private BlazeCommandResult execInternal(CommandEnvironment env, OptionsParsingRe
                 String.format("In repo argument %s: no such repo", e.getKey()),
                 Code.INVALID_ARGUMENTS);
           }
+          if (value instanceof RepoDefinitionValue.SpecialBuiltinRepo) {
+            Path repoPath = ((RepoDefinitionValue.SpecialBuiltinRepo) value).repoPath();
+            String info = String.format("\n## bazel_tools:\nSpecial builtin repo stored at: %s\n\n", repoPath);
+
+            OutputStreamWriter writer = new OutputStreamWriter(
```

Comment:
Instead of printing out the definition right here, can we do it in `showRepo` of https://cs.opensource.google/bazel/bazel/+/master:src/main/java/com/google/devtools/build/lib/bazel/bzlmod/modcommand/ModExecutor.java;l=164?

The advantage is that you can use `printer` there and all repo definitions are printed in the same place. You can add an argument to tell showRepo if it needs to print the special builtin repo definition

Reply:
Yes, it can be done and it looks nicer, thanks. PTAL :)